### ISSUE TITLE
Sethekk Halls ID reset and Anzu Summon

### ISSUE THREAD

**Issue by AgeAgainst** [dalaranwow/dalaran-wow#3730]
**Description**: Sethekk Halls daily heroic reset is not properly resetting Anzu. 

**Current behaviour**: Currently 28 September.  Since 22 September (and possibly before), every other day the heroic ID resets back to 24 hours.  The normal instance bosses reset as normal but Raven's Claw/Essence-Infused Moonstone does not work.  This makes it so you can only summon Anzu every other day. I have tried stealthing directly to the Raven's Claw and killing the boss before it.  You are unable to summon Anzu every day.

**Expected behaviour**: Every 24 hours you should be able to summon Anzu for a chance at the mount.

**Steps to reproduce the problem**:
1. Summon Anzu
2. Kill Anzu
3. Come back next reset to find out you can't summon Anzu again and have to wait an additional day.

http://www.wowhead.com/npc=23035/anzu#comments
Anzu is a summoned boss in a heroic. Do I really need to provide proof that it's supposed to reset every 24 hours?

Anzu: http://www.wowhead.com/npc=23035
Moonstone: http://www.wowhead.com/item=32449

**Comment by AgeAgainst** [comment]
Another day brings an additional 24 hour extension on my Sethekk Halls, making it the second day in a row and four out of the last six days where Anzu has not reset properly.

**Comment by AgeAgainst** [comment]
Another day passes.  The instance resets yet when I kill the first boss I get the same exact raid ID as before (285).  Still unable to summon Anzu.

In the last seven days I have attempted to go into Sethekk Halls to summon Anzu.  Only two days the summoning worked.  Five out of the last seven (including three in a row) has had Anzu in a state where he is unable to be summoned.

What other information do you need before you'll start to investigate this issue?

**Comment by robinsch** [comment]
None, still this is low priority.

**Comment by Rushor** [comment]
there is a internal cooldown for the Infused Moonstone, else you would be able to summon the boss with serveral ids.

**Comment by SadhuArch** [comment]
I'm pretty certain this has nothing to do with the internal cooldown for the Essence-Infused Moonstone. I have been doing Sethekk Hall's the last couple of days without the bug that the instance has extended itself and I have been using the Moonstone in less than 24 hours since the last time I used. I have been clocking it.

However, today the instance had extended itself for another 24 hours, but I decided to try anyway. And as per usual, the only other boss I kill to get to Anzu had respawned but when I try to summon Anzu nothing happens. The cast for "Summon Raven God" is being cast but Anzu is not being summoned. Today is also more than 24 hours since I last summoned him yesterday which suggests that this has nothing to do with an internal cooldown on the Essence-Infused Moonstone.

I might enter the dungeon later today to give it another shot but I'm not feeling hopeful.

**Comment by Rushor** [comment]
if anzu was already killed and that bossstate was saved in any kind of way, he will despawn

**Comment by SadhuArch** [comment]
Then wouldn't that be applied to the first boss Darkweaver Syth as well? Because he respawns whether the instance has been automatically extended or not.

**Comment by AgeAgainst** [comment]
Rushor said:
"there is a internal cooldown for the Infused Moonstone, else you would be able to summon the boss with serveral ids."

People ran this daily after the first druid every completed the quest, and throughout retail TBC and WoTLK in hopes of getting Anzu mount to drop with either their friends or strangers that the druid sold the summon to once they looted the mount.  You are supposed to be able to summon Anzu once per instance lockout.    It wasn't until patch 4.0.1 where Anzu became a permanent boss inside Sethekk Halls, and people continued to run it on a (at least mostly) daily basis.  There was even a hotfix on April 9th, 2009.

http://wowwiki.wikia.com/wiki/Anzu

Hotfix (2009-04-22): "Killing Anzu in Sethekk Halls will properly save you to the instance." 

For timeline reference:
2009-04-19  - Patch 3.1.0
2009-04-22  - Hotfix to make sure stealthing to and only killing Anzu saved you to Sethekk Halls so you would be forced to wait until Sethekk Halls reset to kill him again.
2009-08-04  - Patch 3.2.0

http://www.wowhead.com/forums&topic=127975#p1527281
Comment on wowhead from nitefal, patch 3.2.2:
"My wife and I have been farming this everyday now"

In my previous comment on 28 Sep 2016, I had mentioned that Sethekk Halls had not reset two days in a row, and once again on 29 Sep making it three days in a row.  Surely the internal CD on the Infused Moonstone should have reset by that time, plus with a comment on wowhead of someone saying they farm it daily.  It likely has something to do with the heroic lockout automatically extending itself.  This of course is a wild ass guess on my part, but other data points that the internal CD of the moonstone should not be the issue here if three days in a row Anzu is not able to be summoned.

Yes, I understand that it's a low priority and that there are other issues that have been fixed and other issues getting higher priority.  I get that 100%.  Anzu mount is a vanity item and it isn't game-breaking by any means.  Just trying to provide a bit more information in hopes that once ICC gets released and the more pressing issues get resolved, some progress can be made on this issue.

**Comment by alexuktc** [comment]
I have just started farming the boss. I made a kill 2 days in a row now. I will see if the issue happens to me as well.

**Comment by alexuktc** [comment]
Hello. Like I said I have killed Anzu 6 times in the 6 days since posting my last comment. I have had no issues, I am able to kill Anzu once a day, twice if we count the fact that there is another druid in my guild also farming the boss. We have not had any cooldown problems.

**Comment by Rushor** [comment]
so we can close the issue? any comments from other users to this?

**Comment by SadhuArch** [comment]
Since I started skipping the first boss since I wrote last time a week ago (I used to kill him each time) it hasn't bugged for me.

**Comment by Rushor** [comment]
well then closed for now

**Comment by SadhuArch** [comment]
Today it bugged in the same way again. I was still saved from yesterday and I had not killed Darkweaver Syth (the first boss). I have only been killing Anzu daily since I last wrote here and it has been working fine up until today.

So there IS some kind of bug present here. Whatever it is I can't say, only speculate. And I got the feeling that it has something to do with it being reset day yesterday because the same thing used to happen to me when I was farming Fiery Warhorse's Reins from Attumen in Karazhan.
Whenever I killed Attumen on a Friday, the reset would continue on to the next week ultimately leaving me with a 2 week save to Karazhan.
